backend/eslint.config.mjs

```diff
@@ -1,5 +1,4 @@
-import eslintTsConfig from '../common/eslint-ts.config.mjs';
-import eslintTestConfig from '../common/eslint-test.config.mjs';
+import { eslintTsConfig, eslintTestConfig } from '../eslint-shared.config.mjs';
```

Comment:
I want to spend some time validating this. The eslint config is unfortunately brittle and easy to accidentally cause rules to be applied where they should not be.